stdlib/2and3/ftplib.pyi

```diff
@@ -0,0 +1,123 @@
+                     timeout: float = ..., source_address: Optional[Tuple[str, int]] = ...) -> None: ...
+        def connect(self, host: AnyStr = ..., port: int = ..., timeout: float = ...,
+                    source_address: Optional[Tuple[str, int]] = ...) -> str: ...
+    else:
+        def __init__(self, host: AnyStr = ..., user: str = ..., passwd: str = ..., acct: str = ...,
+                     timeout: float = ...) -> None: ...
+        def connect(self, host: AnyStr = ..., port: int = ..., timeout: float = ...) -> str: ...
+
+    def getwelcome(self) -> str: ...
+    def set_debuglevel(self, level: int) -> None: ...
+    def debug(self, level: int) -> None: ...
+    def set_pasv(self, val: Union[bool, int]) -> None: ...
+    def sanitize(self, s: str) -> str: ...
+    def putline(self, line: str) -> None: ...
+    def putcmd(self, line: str) -> None: ...
+    def getline(self) -> str: ...
+    def getmultiline(self) -> str: ...
+    def getresp(self) -> str: ...
+    def voidresp(self) -> str: ...
+    def abort(self) -> str: ...
+    def sendcmd(self, cmd: str) -> str: ...
+    def voidcmd(self, cmd: str) -> str: ...
+    def sendport(self, host: str, port: int) -> str: ...
+    def sendeprt(self, host: str, port: int) -> str: ...
+    def makeport(self) -> socket: ...
+    def makepasv(self) -> Tuple[str, int]: ...
+    def ntransfercmd(self, cmd: str, rest: Optional[str] = ...) -> Tuple[socket, int]: ...
+    def transfercmd(self, cmd: str, rest: Optional[str] = ...) -> socket: ...
+    def login(self, user: str = ..., passwd: str = ..., acct: str = ...) -> str: ...
+    def retrbinary(self, cmd: str, callback: Callable[[bytes], None], blocksize: int = ..., rest: Optional[str] = ...) -> str: ...
+    def retrlines(self, cmd: str, callback: Optional[Callable[[str], None]] = ...) -> str: ...
+    def storbinary(self, cmd: str, fp: BinaryIO, blocksize: int = ..., callback: Optional[Callable[[bytes], None]] = ..., rest: Optional[str] = ...) -> str: ...
+    def storlines(self, cmd: str, fp: BinaryIO, callback: Optional[Callable[[bytes], None]] = ...) -> str: ...
+    def acct(self, password: str) -> str: ...
+    def nlst(self, *args: str) -> List[str]: ...
+
+    # Technically only the last arg can be a Callable but ...
+    def dir(self, *args: Union[str, Callable[[str], None]]) -> None: ...
+
+    if sys.version_info >= (3, 3):
+        def mlsd(self, path: str = ..., facts: Iterable[str] = ...) -> Iterator[Tuple[str, Dict[str, str]]]: ...
+    def rename(self, fromname: str, toname: str) -> str: ...
+    def delete(self, filename: str) -> str: ...
+    def cwd(self, dirname: str) -> str: ...
+    def size(self, filename: str) -> str: ...
+    def mkd(self, dirname: str) -> str: ...
+    def rmd(self, dirname: str) -> str: ...
+    def pwd(self) -> str: ...
+    def quit(self) -> str: ...
+    def close(self) -> None: ...
+
+class FTP_TLS(FTP):
```

Comment:
If you want this one to also be generic, you need to inherit from `FTP[AnyStr]` (https://www.python.org/dev/peps/pep-0484/#user-defined-generic-types).